client/src/glossary/TooltipActivator.js

```diff
@@ -1,102 +1,12 @@
 import _ from "lodash";
 import React from "react";
-import Tooltip from "tooltip.js";
+import tippy from "tippy.js";
 
 import { get_glossary_item_tooltip_html } from "../models/glossary.js";
 
-// Patch over Tooltip's _scheduleShow and _scheduleHide to not use setTimeout with a 0 second delay
-// The 0 second delay could still result in the _show call being stuck pending for extended periods (was consistently > 1 second on mobile Chrome)
```

Comment:
Glad to see this stuff go! `tippy.js` seems to work fine on my mobile chrome, nice.

I'd forgot all about this hackyness. If I'd remembered I would have redone it using `patch-package`, or thought to switch tooltip libraries sooner, ha.